docs/pages/feature-matrix.mdx

```diff
@@ -13,91 +13,184 @@ ensure that all tables are aligned.*/}
+
+- [Teleport Zero Trust Access](#teleport-zero-trust-access)
+- [Teleport Machine & Workload Identity](#teleport-machine--workload-identity)
+- [Teleport Identity Governance](#teleport-identity-governance)
+- [Teleport Identity Security](#teleport-identity-security)
+
 ## Teleport Zero Trust Access
 
-||[Teleport Enterprise (Cloud)](#teleport-editions)|[Teleport Enterprise (Self-Hosted)](#teleport-editions)|[Teleport Community Edition](#teleport-editions)|
+**Teleport Zero Trust Access** provides engineers with least privileged access
+to applications, servers, databases, Kubernetes clusters, and other resources
+across distributed infrastructures.
+
+**User identity:** Authenticate users without passwords leveraging Touch ID,
+YubiKey Bio and other biometric devices. Add Teleport as a security layer on top
+of your identity provider or use Teleport's SSO.
+
+|  | **Teleport Enterprise (Cloud)** | **Teleport Enterprise (Self-Hosted)** | **Teleport Community Edition** |
 |---|:---:|:---:|:---:|
-|Agentless Integration with [OpenSSH Servers](./enroll-resources/server-access/openssh/openssh-agentless.mdx)|✔|✔|✔|
-|[Dual Authorization](./admin-guides/access-controls/guides/dual-authz.mdx)|✔|✔|✖|
-|[Enhanced Session Recording](./enroll-resources/server-access/guides/bpf-session-recording.mdx)|✔|✔|✔|
-|[FedRAMP Control](./admin-guides/access-controls/compliance-frameworks/fedramp.mdx)|✖|✔|✖|
-|FIPS-compliant binaries available for FedRAMP High|✖|✔|✖|
-|IP-Based Restrictions|✔|✔|✖|
-|[Moderated Sessions](./admin-guides/access-controls/guides/joining-sessions.mdx)|✔|✔|✖|
-|PCI DSS Features|✔|✔|Limited|
-|[Protecting Applications](./enroll-resources/application-access/getting-started.mdx)|✔|✔|✔|
-|[Protecting Databases](./enroll-resources/database-access/getting-started.mdx)|✔|✔|✔|
-|[Protecting Kubernetes Clusters](./enroll-resources/kubernetes-access/getting-started.mdx)|✔|✔|✔|
-|[Protecting Linux Servers](./enroll-resources/server-access/getting-started.mdx)|✔|✔|✔|
-|[Protecting Windows Desktops](./enroll-resources/desktop-access/introduction.mdx)|✔|✔|✔|
-|[Recording Proxy Mode](./enroll-resources/server-access/guides/recording-proxy-mode.mdx)|✖|✔|✔|
-|[Role-Based Access Control](./admin-guides/access-controls/guides/role-templates.mdx)|✔|✔|✔|
-|[Session Recording with Playback](./reference/architecture/session-recording.mdx)|✔|✔|✔|
-|[Single Sign-On](./admin-guides/access-controls/sso/sso.mdx)|GitHub, Google Workspace, OIDC, SAML, Teleport|GitHub, Google Workspace, OIDC, SAML, Teleport|GitHub|
-|SOC 2 Features|✔|✔|Limited|
-|[Structured Audit Logs](./reference/monitoring/audit.mdx)|✔|✔|✔|
+| [Single Sign-On](https://goteleport.com/docs/admin-guides/access-controls/sso/)| GitHub, Google Workspace, Entra, Okta, OIDC, SAML, Teleport | GitHub, Google Workspace, Entra, Okta, OIDC, SAML, Teleport | GitHub |
+| User & Group Provisioning & Deprovisioning (SCIM & Custom Protocols), including Okta and Entra | Available In Teleport Identity Governance | ✖ |
```

Comment:
You're missing a cell here. We should repeat the "Available in..." text for both Cloud and Self-hosted

Reply:
I'll add the list of products to the Self-Hosted column. We don't tend to italicize third-party product names in the docs, so I'm leaning toward keeping these un-italicized for consistency.